src/docs/Components/Notification/code.mdx

```diff
@@ -15,61 +15,48 @@ order: 2
 Wrap a title (it may be any level of heading: `h1`, `h2`, `h3` and so on, depending on the context or SEO requirements), `p` and a `a` or `button` tags in a `div` and apply the `mc-notification` class:
 
 ```html
-  <div class="mc-notification">
-    <div class="mc-notification__content">
-      <h3 class="mc-notification__title">Information notification</h3>
-      <p class="mc-notification__message">
-        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo
-        ligula eget dolor. Aenean massa.
-      </p>
-      <a class="mc-notification__link mc-link" href="#">Link</a>
-    </div>
+<div class="mc-notification">
+  <div class="mc-notification__content">
+    <h3 class="mc-notification__title">Information notification</h3>
+    <p class="mc-notification__message">
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo
+      ligula eget dolor. Aenean massa.
+    </p>
+    <a class="mc-notification__link mc-link" href="#">Link</a>
   </div>
+</div>
 ```
 
 <Preview path="notification" />
 
 ## Types
 
-Available notification types:
- - information (default): `mc-notification--information`
- - success: `mc-notification--success`
- - warning: `mc-notification--warning`
- - danger: `mc-notification--danger`
+Depending on the context of use, you can use different types of notifications.
+
+Mozaic provides you with 4 types of notifications, which you can use with the following css modifiers:
+
+- information (default): `mc-notification--information`
+- success: `mc-notification--success`
+- warning: `mc-notification--warning`
+- danger: `mc-notification--danger`
+
+> The modifier is to be applied on the element having the `mc-notification` class
 
 <Preview path="notification-types" />
 
 ## Variations
 
-Using `mc-notification-closable` class you have a closable notification:
+### Sizes
 
-<Preview path="notification-variations" />
+Mozaic offers you two sizes of notifications that you can use, by applying the following modifiers on the `mc-notification` element :
 
-## Accessibility and semantic
+- Size **S**: use the `mc-notification--s` modifier
+- Size **M**: default size - no modifier needeed
 
-<HintItem>
-  Don't forget to set <code>role="alert"</code> in the notification when It's used to alert about a result of user action
-</HintItem>
+<Preview path="sizes" />
```

Comment:
The icon looks more thin? Is this a 24px icon?

Reply:
Fixed